coderefs/coderefs.go

```diff
@@ -68,7 +68,9 @@ func Run(opts options.Options) {
 		CommitTime:       commitTime,
 	}
 
-	handleOutput(opts, matcher, branch, repoParams, ldApi)
+	if output {
+		handleOutput(opts, matcher, branch, repoParams, ldApi)
+	}
```

Comment:
i understand that this is a function name that already existed, but it was reaaaaaally confusing to me for a few minutes how this enabled us to run just extinctions 😅  - can we rename it? Like, "saveReferences" or something?

Reply:
@jazanne how about something like `generateHunkOutput` for naming? We should probably split this down more in the future and it's a relatively recent function, for right now though it does manage if it's writing to `csv` or `POST` to API so #namingthingsishard

Reply:
yea that works for me 👍